### DISCUSSION TITLE
Error Starting (version 1.0.0)

### DISCUSSION THREAD

**Discussion by Praveenstein** [apache/iotdb#8781] [Q&A] [answered]
I was able to run the previous version (v0.13), but new version there is some error, please help me out
![Untitled](https://user-images.githubusercontent.com/45847217/211149159-d3389c84-9285-4f1e-9b67-3e7651534a94.png)

**Comment by xiaoyekanren** [comment, marked as answer]
I found a "space" in your path, "apache iotdb"
I suggest rename "apache iotdb" to "apache_iotdb" or "apache-iotdb", then restart again. 

<img width="1025" alt="image" src="https://user-images.githubusercontent.com/7812383/211274521-af9747b0-e98e-4e98-81ff-97f1a146a8aa.png">

**Reply by Praveenstein** [reply]
Thanks worked like a charm!
